src/README.md

```diff
@@ -12,9 +12,27 @@ visits the website. Please refer to `js/index.js` for how these values
 are generated.
 
 Avoid changing the "Current Release Versions" without also updating the selector
-in `js/index.js.
+in `js/index.js`.
 -->
```

Comment:
Can you move this comment so it's directly above the current release versions text?